lib/modules/hover.js

```diff
@@ -203,6 +204,7 @@ class Hover {
 		const $container = this.getContainer();
 
 		if (this._callback) {
+			if (this._displayLoadIndicator) this._displayLoadIndicator($container);
```

Comment:
Nit: make `_displayLoadIndicator` an empty method on `Hover` and remove the null-check (making it more obvious that this is something that a child class can/should override)

Also, _maybe_ call it `_onOpen` or `_beforeOpen` or something (i.e. why/when it's called rather than what the child class should do). Though I'm not sure if that's overly generic, since we'll probably never use it for anything but a loading icon, so feel free to leave it as-is.